tests/unit/btree_stress_test.c

```diff
@@ -113,7 +113,7 @@ CTEST_DATA(btree_stress)
 // Setup function for suite, called before every test in suite
 CTEST_SETUP(btree_stress)
 {
-   set_log_streams_for_error_tests(NULL, NULL);
+   set_log_streams_for_tests(MSG_LEVEL_ERRORS);
```

Comment:
First, this is a bug-fix to use the proper interface.

Second, the way to understand this call is that: The test is saying that: "I want to see output ONLY when the output logging-level is MSG_LEVEL_ERRORS (i.e., 3)".

Maybe I could change the check in `set_log_streams_for_tests` to check for:

   `if (Ctest_verbosity >= exp_msg_level)`
   
... But I wanted to have the reviewers give their opinion before going down that route.

Reply:
I'd describe the meaning of this call differently.  It isn't about the test saying what output it wants to see.  It's about the test case declaring that it will be logging messages to the log-stream at a given severity (or at a more severe / lower number).  Therefore, the test-runner ought to have the log-stream map to stdout if and only if the user has requested to see messages at that severity (or less severe / higher number).  Otherwise, it should map to /dev/null.

Ideally, that logic would be on a per-call basis.  But since that would be a lot more work than we want to do right now, we instead have it at the granularity of per-test-case.

Reply:
Yeah, given the current implementation, this argument needs to be `MSG_LEVEL_DEBUG`.  Because on the current commit, 
```
VERBOSE=1 build/release/bin/unit/btree_stress_test
```
gives all the debug output.

Reply:
Fixed the bug in `btree_stress_test.c`. Thanks for catching it.

Also took the opportunity to add a prolog comment to `set_log_streams_for_tests()`, to help future authors. 

Let me know if you wish to wordsmith that even more.